docs2/site/docs/getting-started/custom-scalars.md

```diff
@@ -48,16 +54,12 @@ mutation AddVector($vector3: Vector3!) {
 }
 ```
 
-Vectors should be received in a more structured format:
+Vectors should be returned in the same format:
 
 ```json
 {
     "data": {
-        "getVector": {
```

Comment:
Strange. Is it a scalar? This is an object. Documentation was incorrect?

Reply:
No, the documentation was demonstrating how to have a scalar return a complex object.  I moved the demonstration further down the page, after a more basic demonstration.